dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,35 @@
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
```

Comment:
It doesn't delete volumes, it just stops and removes all containers (which is throw-away, recreated) and network (which is throw-away, recreated).

Basically, `docker-stop` will undo what `docker-start` did, except volumes and images themselves.